docs/book/beyond_basics/boilerplate.md

```diff
@@ -1,24 +1,18 @@
-{% panel style="info", title="Under Development" %}
-This book is being actively developed.
-{% endpanel %}
-
 # Defining Boilerplate License Headers
 
 {% method %}
 
 It is possible to add boilerplate license headers to all generated code by
-modifying `hack/boilerplate.go.txt`.
+defining `hack/boilerplate.go.txt` before initializing a project.
 
-If you don't create `boilerplate.go.txt` an empty version will be created for you by
-`kubebuilder init`.  Modifying this file will only impact files created afterward.
+If you don't create `boilerplate.go.txt` an apache2 boilerplate header before
+running init an apache2 header will be created for you by default.
```

Comment:
If you don't create `boilerplate.go.txt` before running init, an apache2 header will be ...
or
If you don't create `boilerplate.go.txt`, an apache2 header will be ...